src/components/atoms/VisualisationParameterSelector.tsx

```diff
@@ -44,20 +31,13 @@ const VisualisationParameterSelector: React.FC<VisualisationParameterSelectorPro
                 <option value="medium">Medium</option>
                 <option value="large">Stor</option>
             </SelectField>
-            <Heading size={400} marginBottom="1rem">
-                Legg til tekst
-            </Heading>
-            <Pane display="flex" marginBottom="1rem">
-                <Switch checked={withText} onChange={handleSwitchChange} marginRight="0.5rem" />
-                {!withText && (
-                    <Text size={300} userSelect="none">
-                        Ingen tekst
-                    </Text>
-                )}
-            </Pane>
+            <Label htmlFor="textarea" marginBottom={12}>
+                Legg til text
+            </Label>
             <Pane flex="1">
                 <Textarea
-                    disabled={!withText}
+                    id="textarea"
```

Comment:
Perhaps attempt to use more descriptive IDs. This could quickly cause collisions:)

Reply:
true

Reply:
!!false

Reply:
true + true === 2